src/main/scripts/src/app/core/error/lumeer-error.handler.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *
+ * Copyright (C) since 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+
+import {ErrorHandler} from '@angular/core';
+
+/**
+ * This class provides handling unexpecting errors
+ */
+export class LumeerErrorHandler implements ErrorHandler {
+
+  public handleError(error: any): void {
+    alert('An unexpected error occured');
```

Comment:
Of course that no. I would rather use component I sent to the slack, but you and @marvec must approve it.